### DISCUSSION TITLE
prop `classNames` will not tirgger vscode tailwindcss intellisense.

### DISCUSSION THREAD

**Discussion by Innei** [heroui-inc/heroui#1666] [Help]
Tailwind Intellisense is not working.

![CleanShot 2023-09-21 at 5  54 44@2x](https://github.com/nextui-org/nextui/assets/41265413/dd9e5121-9b17-41fd-9867-18cede4df334)

**Comment by JCYDigitalinnov** [comment]
I think this will help. @Innei 

1. Go to your `settings.json`
![image](https://github.com/nextui-org/nextui/assets/118256336/8e12a921-c44d-4605-88c0-08d02a9cb039)
2. Just add the `classNames` under the `tailwindCSS.classAttributes`
3. Then it will work now.
![image](https://github.com/nextui-org/nextui/assets/118256336/84248200-6bba-44e2-8d46-6e40bb720a2e)

**Reply by notmedia** [reply]
@jrgarciadev Please add this useful tip to docs
